CONTRIBUTING.md

```diff
@@ -27,7 +27,26 @@ Encountering challenges is a normal part of the process. If you need help:
 ## Contributing Guidelines
 
 - **Commit Messages:** Use concise and descriptive commit messages. This helps with the project's history and review process.
-- **Pull Requests:** Before submitting a pull request, make sure your code complies with the project's standards and passes all tests by running `dune fmt` and `dune runtest`. Note you will need `ocamlformat=0.26.1`.
+- **Pull Requests:** Before submitting a pull request, make sure your code complies with the project's standards and passes all tests.. This process involves two key steps:
+  1. **Code Formatting:** Use `dune fmt` to automatically format your code according to the project's style guidelines. This command will apply the formatting rules defined in the `.ocamlformat` configuration file, ensuring consistency across the project.
+
+      To run this command, navigate to the root directory of your project in your terminal and execute:
+       
+            dune fmt
+      
+      **Note:** You will need `ocamlformat=0.26.1` installed to format your code correctly. If you haven't installed it yet, you can do so by running:
+          
+          opam install ocamlformat.0.26.1
+      
+      To learn more about `dune fmt`, check out the Man page [here](https://www.mankier.com/1/dune-fmt), or run `man dune fmt` on your terminal.
```

Comment:
The man page is not very interesting as it contains very little documentation and a lot of unrelated options. The documentation is better: https://dune.readthedocs.io/en/stable/howto/formatting.html#running-the-formatters

Reply:
Thanks for this.